common/council_positions/00_council_positions.txt

```diff
@@ -0,0 +1,2289 @@
+councillor_chancellor = {
+	skill = diplomacy
+	
+	name = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					is_male = yes
+				}
+				desc = {
+					triggered_desc = {
+						trigger = {
+							scope:councillor_liege.culture = culture:old_saxon
+						}
+						desc = chancellor_male_old_saxon
+					}
+					triggered_desc = {
+						trigger = {
+							scope:councillor_liege.culture = culture:welsh
+						}
+						desc = chancellor_male_welsh
+					}
+					triggered_desc = {
+						trigger = {
+							scope:councillor_liege.culture = culture:cornish
+						}
+						desc = chancellor_male_cornish
+					}
+					triggered_desc = {
+						trigger = {
+							scope:councillor_liege.culture = culture:cumbric
+						}
+						desc = chancellor_male_cumbric
+					}
+					triggered_desc = {
+						trigger = {
+							scope:councillor_liege.culture = culture:portuguese
+						}
+						desc = chancellor_male_portuguese
+					}
+					triggered_desc = {
+						trigger = {
+							scope:councillor_liege.culture = culture:greek
+						}
+						desc = chancellor_male_greek
+					}
+					triggered_desc = {
+						trigger = {
+							scope:councillor_liege.culture = culture:italian
+						}
+						desc = chancellor_male_italian
+					}
+					triggered_desc = {
+						trigger = {
+							scope:councillor_liege.culture = culture:roman
+						}
+						desc = chancellor_male_roman
+					}
+					triggered_desc = {
+						trigger = {
+							OR = {
+								scope:councillor_liege.culture = culture:bosian
```

Comment:
1)Is it really bosian not bosnian in the files?
2)Independently, doesn't it make more sense to mark thing which are common to existing Serbian and anachronistic Bosnian "serbian"?

Reply:
I followed this up for the female section.